### DISCUSSION TITLE
OTA Update NVIDIA Jetson Using Allxon | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#847] [Q&A]
# OTA Update NVIDIA Jetson Using Allxon | Seeed Studio Wiki

Update Jetson Linux Over-the-Air Using Allxon

https://wiki.seeedstudio.com/Update-Jetson-Linux-OTA-Using-Allxon/

**Comment by KamoliddinS** [comment]
hi, I got confused, which part we have to run and prepare OTA Payload in Host machine, which part is for our jetson device
